.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+---
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
```

Comment:
ooh the version in gnomad methods is v2.3.0, should we update that?

Reply:
Might as well